sphinx/tutorial/cylc/furthertopics/clock-triggered-tasks.rst

```diff
@@ -0,0 +1,179 @@
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+Clock Triggered Tasks
+=====================
+
+This part of the Rose user guide walks you through using clock triggered tasks.
+
+This allows you to trigger tasks based on the actual time.
```

Comment:
I suggest merging this second introductory sentence with the first via 'tasks, which allow you...' to create a helpful self-contained introductory sentence and avoid an initial comment stating the obvious and hesitation over what 'this' refers to, as advocated for the equivalent prelude in the 'cylc broadcast' file.

Reply:
This is all a bit clunky, it's just the way the old tutorials were written, some I re-wrote, others I left. I might go through and address these ...

Reply:
Sounds like a good idea. Rest assured I will stop making this same comment!